packages/realm-web/src/Credentials.ts

```diff
@@ -53,10 +53,11 @@ type SimpleObject = Record<string, unknown>;
 export class Credentials<PayloadType extends SimpleObject = SimpleObject> implements Realm.Credentials<PayloadType> {
   /**
    * Creates credentials that logs in using the [Anonymous Provider](https://docs.mongodb.com/realm/authentication/anonymous/).
+   * @param reuse - Reuse any existing anonymous user already logged in.
    * @returns The credentials instance, which can be passed to `app.logIn`.
    */
-  static anonymous(): Credentials<AnonymousPayload> {
-    return new Credentials<AnonymousPayload>("anon-user", "anon-user", {});
+  static anonymous(reuse = true): Credentials<AnonymousPayload> {
```

Comment:
Sidenote: For both realm and realm-web, perhaps a future overload (or replacement) with an options object would make the calling code a bit more readable 🙂

Reply:
Once reason I choose this is to align with the current API of the `realm` package:

https://github.com/realm/realm-js/blob/0bd8ee486780431594724e3ff4988aa0853e7a4e/packages/realm/src/app-services/Credentials.ts#L56

Reply:
Yup I figured 👍